docker-compose.yml

```diff
@@ -4,8 +4,6 @@ services:
     build: .
     ports:
       - "5000:5000"
-    # Wait for db container to finish setting up postgres
-    command: bash -c 'while !</dev/tcp/db/5432; do sleep 1; done; ./scripts/entrypoint.sh'
```

Comment:
Only the dev environment uses this so and it was causing an CI issue so I moved it to the dev specific `docker-compose.dev.yml` file